src/core/propertyTypes.js

```diff
@@ -38,10 +39,20 @@ function registerPropertyType (type, defaultValue, parse, stringify) {
 }
 module.exports.registerPropertyType = registerPropertyType;
 
+function arrayParse (value) {
+  return value.length === 0 ?
```

Comment:
you should check if the value is already an array to see if you need to parse at all